steve/util.py

```diff
@@ -378,10 +378,14 @@ def out(*output, **kw):
     sys.stdout.write(output + '\n')
 
 
-def load_json_files(config):
+def load_json_files(config, path=None):
```

Comment:
Mmm... I think path is overloaded here. It's probably better to have the `steve.ini` file mark the project root directory and have a `rootdir` parameter in the config which we can then use to figure out the rest of it.